lib/github/middleware.js

```diff
@@ -1,9 +1,17 @@
 const { Subscription } = require('../models')
 const getJiraClient = require('../jira/client')
 const getJiraUtil = require('../jira/util')
+const { Sentry } = require('../error-handler')
 
 module.exports = function middleware (callback) {
   return async (context) => {
+    Sentry.setContext('GitHub Payload', {
+      event: context.name,
+      action: context.payload.action,
+      id: context.id,
+      nwo: (context.payload.repository) ? context.repo() : undefined
```

Comment:
What does `nwo` stand for?

Reply:
It's "Name With Owner", which is a general abbreviation I see used when we want to log the repo name with it's owner. We could probably just call it `repo` though. It returns an object that has both the repo name and owner login, so while it has more data than just the repo, maybe it's less confusing than calling it `nwo`, especially since the function is `.repo()`.

Reply:
So `integrations/jira` instead of `jira`?

I like `repo` for the reasons you've outlined. 💯

Reply:
> So integrations/jira instead of jira?

Yeh exactly. Since a repo name by itself isn't enough data.

I updated to `repo` in 83fd55fe83ec1ceb271f778bf6a978e4be74bc87